FRW/Services/Manager/WalletManager.swift

```diff
@@ -796,6 +800,41 @@ extension WalletManager {
         }
     }
 
+    func fetchAccountInfo() async throws {
+        do {
+            let accountInfo = try await FlowNetwork.checkAccountInfo()
+            await MainActor.run {
+                self.accountInfo = accountInfo
+            }
+        } catch let error {
+            log.error("[EVM] fetch account info failed.\(error.localizedDescription)")
```

Comment:
It's not EVM action, let's put other tag. Such as [WALLET]

Reply:
A clear case of copy and paste from another line :-)